l3.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+
+waluty = ['ZEC', 'LTC', 'DASH']
+url1 = "https://bitbay.net/API/Public/"
+url2 = "USD/ticker.json"
+
+
+def downloadData(currency, url1, url2):
+    url = url1 + currency + url2
+    data = requests.get(url).json()
+    return data['ask'], data['bid']
+
+
+def plotData(plt1, plt2, plt3):
+    ask_BTC, bid_BTC = downloadData(waluty[0], url1, url2)

```

Comment:
trochę bardziej generycznie, przy 50 parach miałbyś to 100 linii kodu?;)

Reply:
po to masz zdefiniowaną waluty (aka ASSETS  / CURRENCIES), żeby móc po niej iterować